### PR TITLE
docs: Add Confluent.Kafka to the .NET Agent compatibility list.

### PR DESCRIPTION
**Pull request by tippmar-nr**
## Give us some context

Adds `Confluent.Kafka` to the list of messaging systems supported by the .NET Agent.

### CODE REVIEW

File: src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core.mdx [hunk 1]
```diff
@@ -666,6 +666,25 @@ Use [Elastic.Clients.Elasticsearch](https://www.nuget.org/packages/Elastic.Clien
       </thead>
 
       <tbody>
+        <tr>
+          <td>
+            Confluent.Kafka
+          </td>
+
+          <td>
+            * Produce and Consume on Topics.
```
Comment (ally-sassman) [inline src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core.mdx:675]: @tippmar-nr Can you clarify what this bullet point means?

Comment (tippmar-nr) [reply src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core.mdx:675]: These are the type of actions ("produce" and "consume") we instrument in Kafka.

[the remark this answers is not in the document]
Comment (tippmar-nr) [reply src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core.mdx:675]:
```suggestion
            * Produce and consume on topics.
```
Should be sufficient - looks like doing so would make us consistent across this doc.

File: src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx [hunk 2]
```diff
@@ -903,6 +903,25 @@ Use [Elastic.Clients.Elasticsearch](https://www.nuget.org/packages/Elastic.Clien
       </thead>
 
       <tbody>
+        <tr>
+          <td>
+            Confluent.Kafka
+          </td>
+
+          <td>
+            * Produce and Consume on Topics.
```
Comment (tippmar-nr) [inline src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx:912]:
```suggestion
            * Produce and consume on topics.
```

**Review by ally-sassman — APPROVED** [review]

**Pull request merged**